### PR TITLE
WIP Dbz 4392 specify maven as source for connector artifacts

### PR DESCRIPTION
**Pull request by roldanbob**
Added changes for Streams deployment.
Todo: `curl` command in Dockerfile for legacy deployment

### CODE REVIEW

**Comment by Naros** [comment]
@roldanbob feel free to ping me when all the work on this PR is ready for review.

File: documentation/modules/ROOT/pages/connectors/db2.adoc [hunk 1]
```diff
@@ -2155,13 +2139,6 @@ The following example sets the message key for the tables `inventory.customers`
  +
 In the preceding example, the columns `pk1` and `pk2` are specified as the message key for the table `inventory.customer`.
 For `purchaseorders` tables in any schema, the columns `pk3` and `pk4` serve as the message key.
-
-|[[db2-property-schema-name-adjustment-mode]]<<db2-property-schema-name-adjustment-mode,`+schema.name.adjustment.mode+`>>
-|avro
-|Specifies how schema names should be adjusted for compatibility with the message converter used by the connector. Possible settings:  +
-
-* `avro` replaces the characters that cannot be used in the Avro type name with underscore. +
-* `none` does not apply any adjustment. +
```
Comment (Naros) [inline documentation/modules/ROOT/pages/connectors/db2.adoc:2164]: I don't believe this should have been removed as this was added as a part of DBZ-3535 recently.

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 2]
```diff
@@ -2589,13 +2564,6 @@ Hashing strategy version 2 should be used to ensure fidelity if the value is bei
 |bytes
 |Specifies how binary (`blob`) columns should be represented in change events, including: `bytes` represents binary data as byte array (default), `base64` represents binary data as base64-encoded String, `hex` represents binary data as hex-encoded (base16) String
 
-|[[oracle-property-schema-name-adjustment-mode]]<<oracle-property-schema-name-adjustment-mode,`+schema.name.adjustment.mode+`>>
-|avro
-|Specifies how schema names should be adjusted for compatibility with the message converter used by the connector. Possible settings:  +
-
-* `avro` replaces the characters that cannot be used in the Avro type name with underscore. +
-* `none` does not apply any adjustment. +
-
```
Comment (Naros) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:2598]: This should not have been removed.

File: documentation/modules/ROOT/partials/modules/all-connectors/obs_ref-connector-incremental-snapshot.adoc [hunk 3]
```diff
@@ -1,116 +0,0 @@
-[[{context}-adhoc-snapshot]]
-
-= Ad-hoc snapshot
-
```
Comment (Naros) [inline documentation/modules/ROOT/partials/modules/all-connectors/obs_ref-connector-incremental-snapshot.adoc:4]: This file is still references by `mysql.adoc` and others to my knowledge in this PR, so why is this file being randomly removed?

Comment (roldanbob) [reply documentation/modules/ROOT/partials/modules/all-connectors/obs_ref-connector-incremental-snapshot.adoc:4]: This is an older version of the file that I replaced when I was working on the incremental snapshots feature and that I overlooked when cleaning up. The content referenced from other files is now in  `con-connector-incremental-snapshot.adoc`.

File: documentation/modules/ROOT/pages/connectors/db2.adoc (hunk 1 above)
Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/db2.adoc:2164]: I didn't remove this intentionally. My repo must be slightly out of synch and didn't pick up the change. Should I just re-add it and push the change?

File: documentation/modules/ROOT/pages/connectors/oracle.adoc (hunk 2 above)
Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:2598]: I didn't remove this intentionally. Should I re-add the content and push the change?

File: documentation/modules/ROOT/pages/connectors/db2.adoc (hunk 1 above)
Comment (Naros) [reply documentation/modules/ROOT/pages/connectors/db2.adoc]: You could rebase your changes based on `main` as this should be part of `main` to get it.  If that doesn't work then yes please re-add.

**Review by Naros — APPROVED** [review]

**Pull request merged**